sql/telemetry_derived/addon_aggregates_v2/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+friendly_name: Addon Aggregates
+description: >-
+  Addon usages by clients, aggregated across unique sets of dimensions
+  and partitioned by day
+owners:
+  - bmiroglio@mozilla.com
+labels:
+  application: addons
+  schedule: daily
+  incremental: true
+scheduling:
+  dag_name: bqetl_addons
+  depends_on:
+    - dag_name: main_summary
+      task_id: copy_deduplicate_main_ping
```

Comment:
I think it would be worth at this point adding support in the dag generation machinery for identifying dependencies on stable tables and injecting these dependencies on copy_dedupe tasks automatically, so that we can avoid `depends_on` entries. We're definitely going to accidentally miss adding `depends_on` at some point.

To be clear, this is a suggested follow-up. I opened https://github.com/mozilla/bigquery-etl/issues/1089 for this.